content/3_Silver/Binary_Search.mdx

```diff
@@ -82,7 +82,7 @@ export const problems = {
 			'Plat',
 			'Load Balancing',
```

Comment:
For this problem, we should provide a detemplified solution (the template is too long :stuck_out_tongue: ). Could you detemplify or should someone else do it?

Reply:
you can do it :)